packages/teams-js/src/public/interfaces.ts

```diff
@@ -530,7 +530,7 @@ export interface Context {
    * @deprecated
    * As of 2.0.0, please use {@link app.UserInfo.id | app.Context.user.id} instead
    *
-   * The Azure AD object id of the current user.
+   * The Microsoft Entra object id of the current user.
```

Comment:
ID

Reply:
I think this means to change `object id` to `object ID`